2.1.4/buster/build-time-pip-constraints.txt

```diff
@@ -163,7 +163,7 @@ cached-property==1.5.2
 cachetools==4.2.2
 cassandra-driver==3.25.0
 cattrs==1.5.0
-celery==4.4.7
+celery==5.2.3
```

Comment:
We'd need to backport this too:
https://github.com/apache/airflow/pull/17397

And part of why we [ignore the celery upgrade already](https://github.com/astronomer/ap-airflow/blob/b9729541a9ec478e0bc975bb91c7a4d7e715fddf/2.1.4/buster/trivyignore#L12).

Reply:
Yeah I was worried that might happen.

Repackaging the provider might be easier?

Reply:
Yeah I was worried that might happen.

Repackaging the provider might be easier?